zenodo/config.py

```diff
@@ -1403,3 +1409,11 @@ def _(x):
 
 # Error template
 THEME_429_TEMPLATE = "zenodo_errors/429.html"
+
+# Precached thumbnails
+PRECACHED_THUMBNAILS = {
```

Comment:
```suggestion
CACHED_THUMBNAILS = {
```

It's kind of like the argument about "pre-reserving" DOIs